lib/modules/dosomething/dosomething_reportback/dosomething_reportback.forms.inc

```diff
@@ -53,26 +54,76 @@ function dosomething_reportback_form($form, &$form_state, $entity = NULL) {
       '#default_value' => $username,
     );
   }
+
+
+  // Reportback Submissions container.
+  $form['reportback_submissions'] = array(
+    '#type' => 'container',
+    '#id' => 'reportback-submissions',
+    '#attributes' => array(
+      'class' => array(
+        'reportback__submissions',
+      ),
+    ),
+  );
+
+  // @TODO - Remove this switch once we get rid of the old experience.
+  $reportback_file_class = (theme_get_setting('show_new_reportback')) ? 'js-image-upload-beta' : 'js-image-upload';
+  $form['reportback_submissions']['reportback_file'] = array(
+    '#weight' => 5,
+    '#type' => 'file',
+    '#theme_wrappers' => array(),
+    '#attributes' => array(
+      'class' => array(
+        $reportback_file_class
+      ),
+    ),
+  );
+
   // If rbid doesn't exist, this is a create form.
   if (!isset($entity->rbid)) {
     $entity->rbid = 0;
+
+    $form['reportback_submissions']['reportback_file']['#attributes']['is_supersized'] = TRUE;
+
     $submit_label = t("Submit your pic");
   }
-  // Else, it's update form.
+
+  // Else, it's update an form.
```

Comment:
Lol. I had it fixed correctly before and then Vagrant crapped out on me and I had to redo some code. Put the 'an' in the wrong place, haha.